### PR TITLE
Add SVG results

### PR DESCRIPTION
**Pull request by sainirmayi**
## Describe your changes
Preview: https://feature-no-ref-add-svg-results--openproblems.netlify.app/results/spatially_variable_genes/

## Issue ticket number and link
Closes #xxxx (Replace xxxx with the GitHub issue number)

## Checklist before requesting a review
- [x] I have performed a self-review of my code

- Check the correct box. Does this PR contain:
  - [ ] Breaking changes
  - [x] New functionality
  - [ ] Major changes
  - [ ] Minor changes
  - [ ] Bug fixes

- [ ] Proposed changes are described in the CHANGELOG.md

- [ ] CI Preview succeeds and looks good!

### CODE REVIEW

**Comment by KaiWaldrant** [comment]
Found a typo in metric description: https://github.com/openproblems-bio/openproblems-v2/pull/462/files#r1716661275

**Comment by KaiWaldrant** [comment]
Maybe also update the datasets page by running https://github.com/openproblems-bio/openproblems-v2/blob/4e1c139810ed6b0c4f90c9daf976bff9c0c68ac9/src/common/process_dataset_metadata/run/run.sh

File: results/_include/_task_template.qmd [hunk 1]
```diff
@@ -86,3 +86,36 @@ cat(
...
+    roles <- author$roles %>% paste(collapse = ", ")
+    github <- author$info$github %||% NULL
+    orcid <- author$info$orcid %||% NULL
+
+    links <- c()
+    if (!is.null(github)) {
+      links <- c(links, glue::glue("[GitHub](https://github.com/{github})"))
+    }
+    if (!is.null(orcid)) {
+      links <- c(links, glue::glue("[ORCID](https://orcid.org/{orcid})"))
```
[the remark this answers is not in the document]
Comment (sainirmayi) [reply results/_include/_task_template.qmd:105]: Yes, icons would be better.

**Comment by lzj1769** [comment]
@rcannood 

Can you review this request? :)

**Comment by rcannood** [comment]
I did review it -- internally. 

This PR looks good to me! 

I'm wondering whether you could still polish the task info:

* Could you shorten the summary?  Perhaps something like: `Detecting genes whose expression levels vary across spatial regions` already covers the general purpose quite well
* Right now the thumbnail contains an actual plot of gene expression. I'm wondering whether:
  - you could smoothen the surface to better fit with the other task images
  - the diagram on the left currently doesn't match the graph shown on the right. could you make sure that the two match?
  - The image doesn't have the right aspect ratio in comparison to the other tasks.

Aside from that, everything looks good to me! Since there are 50 datasets the funkyheatmap doesn't look very nice. I've discussed with Nirmayi that we need to update the task info and website such that the task should be able to specify how to aggregate the results into different columns. In this task, this could be by technology and by tissue.

**Comment by lzj1769** [comment]
@rcannood 

Sure, I have updated the summary as you suggested.

The thumbnail's surface has been smoothed, and now the image has the same aspect ratio as others.

Because the gene expression was simulated to better control the smoothness and spatial variability, it didn't precisely match the tissue structure shown on the left. Since this is only used to demonstrate the concept of spatial variation, it should be ok.

It sounds good to me to aggregate the results.

**Comment by rcannood** [comment]
@lzj1769 thanks a lot!

@sainirmayi could you update this pr? :bow:

**Pull request merged**